src/core/component.ts

```diff
@@ -61,12 +61,14 @@ const BaseComponent: VueConstructor = Vue.extend({
             };
         },
 
-        $_fillTemplate(template: any): object {
+        $_fillTemplate(slots: Record<string, any>, name: string): object {
             return {
                 render: (data: any) => {
                     const vm = new Vue({
+                        name,
                         parent: this,
                         render: () => {
+                            const template = slots[name];
```

Comment:
Is it really necessary to retrieve template in the callback?
If not it's better just add the second param without modifying the first one: ` $_fillTemplate(template: any, string name)`